tools/scripts/lint/index.test.js

```diff
@@ -16,40 +16,48 @@ describe('markdown linter', () => {
     process.exitCode = 0;
   });
 
-  it('should pass `good` markdown', done => {
-    function callback() {
-      expect(process.exitCode).toBe(0);
-      done();
-    }
-    lint(good, callback);
+  it('should pass `good` markdown', async () => {
+    await new Promise(resolve => {
+      function callback() {
+        expect(process.exitCode).toBe(0);
+        resolve();
+      }
+      lint(good, callback);
+    });
   });
 
-  it('should fail invalid YML blocks', done => {
-    function callback() {
-      expect(process.exitCode).toBe(1);
-      done();
-    }
-    lint(badYML, callback);
+  it('should fail invalid YML blocks', async () => {
+    await new Promise(resolve => {
+      function callback() {
+        expect(process.exitCode).toBe(1);
+        resolve();
+      }
+      lint(badYML, callback);
+    });
   });
 
-  it('should fail when code fences are not surrounded by newlines', done => {
-    function callback() {
-      expect(process.exitCode).toBe(1);
-      done();
-    }
-    lint(badFencing, callback);
+  it('should fail when code fences are not surrounded by newlines', async () => {
+    await new Promise(resolve => {
+      function callback() {
+        expect(process.exitCode).toBe(1);
+        resolve();
+      }
+      lint(badFencing, callback);
+    });
   });
 
-  it('should write to the console describing the problem', done => {
-    function callback() {
-      const expected =
-        'badYML.md: 19: yaml-linter YAML code blocks should be valid [bad indentation of a mapping entry at line 3, column 17:\n          testString: testString\n                    ^] [Context: "```yml"]';
-      expect(console.log.mock.calls.length).toBe(1);
-      expect(console.log.mock.calls[0][0]).toEqual(
-        expect.stringContaining(expected)
-      );
-      done();
-    }
-    lint(badYML, callback);
+  it('should write to the console describing the problem', async () => {
+    await new Promise(resolve => {
+      function callback() {
+        const expected =
+          'badYML.md: 19: yaml-linter YAML code blocks should be valid [bad indentation of a mapping entry at line 3, column 17:\n          testString: testString\n                    ^] [Context: "```yml"]';
+        expect(console.log.mock.calls.length).toBe(1);
+        expect(console.log.mock.calls[0][0]).toEqual(
+          expect.stringContaining(expected)
+        );
+        resolve();
+      }
+      lint(badYML, callback);
+    });
```

Comment:
```suggestion
    await new Promise(resolve => lint(badYML, resolve));

    const expected =
      'badYML.md: 19: yaml-linter YAML code blocks should be valid [bad indentation of a mapping entry at line 3, column 17:\n          testString: testString\n                    ^] [Context: "```yml"]';
    expect(console.log.mock.calls.length).toBe(1);
    expect(console.log.mock.calls[0][0]).toEqual(
      expect.stringContaining(expected)
    );
```